src/main/java/frc/robot/Constants.java

```diff
@@ -16,4 +16,70 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+
+    // ADD THE PORT NUMBERS TO CONSTANTS!!!!!
+
+    public static final class DriveConstants {
+        public static final int kDriveMotor1 = 0;
```

Comment:
Change these to kLeftDriveMotor1Port, etc.  We need to know which side is which.